linux/guest_customization/linux_gosc_workflow.yml

```diff
@@ -82,6 +82,31 @@
             package_state: "present"
           when: guest_os_ansible_distribution != "VMware Photon OS"
 
+        # Need to install the latest cloud-init to Oracle Linux 8.7, 9.1 and pass the GOSC tests in Ansible
+        - name: "Install latest 'cloud-init' for cloud-init GOSC on {{ guest_os_ansible_distribution }}"
+          block:
```

Comment:
You didn't see my comment clearly. The known issue shall be put in this block as the first task. And it is nothing about UEK kernel, it is about cloud-init. You can describe the message as:

```
msg:
  - "Cloud-init guest customization would fail because cloud-init-local.service couldn't start when using cloud-init 22.1-5.0.1."
  - "The issue has been resolved in cloud-init 22.1-6.0.1. Here will upgrade cloud-init to latest version for resolution."
  - "Please refer to https://bugzilla.oracle.com/bugzilla/show_bug.cgi?id=18115 for details."
```

Reply:
Hi Qi,

Updated and it runs OK now.

ok: [localhost] => {
    "msg": [
        "Cloud-init guest customization would fail because cloud-init-local.service couldn't start when using cloud-init 22.1-5.0.1.",
        "The issue has been resolved in cloud-init 22.1-6.0.1. Here will upgrade cloud-init to latest version for resolution.",
        "Please refer to https://bugzilla.oracle.com/bugzilla/show_bug.cgi?id=18115 for details."
    ]
}

If have any other problem, please kindly tell me.

Thanks!
Yanhui